src/Microsoft.Health.Dicom.Core/Features/Retrieve/IFrameHandler.cs

```diff
@@ -11,5 +11,5 @@ namespace Microsoft.Health.Dicom.Core.Features.Retrieve;
 
 public interface IFrameHandler
 {
-    Task<IReadOnlyCollection<Stream>> GetFramesResourceAsync(Stream stream, IEnumerable<int> frames, bool originalTransferSyntaxRequested, string requestedRepresentation);
+    Task<IReadOnlyCollection<Stream>> GetFramesResourceAsync(Stream stream, IEnumerable<int> frames, bool originalTransferSyntaxSelected, string requestedTransferSyntax);
```

Comment:
Changed the names of these parameters to clarify intent - at this point, the `RetrieveResourceService` has evaluated the request, and selected to return the original transfer syntax.